pkg/webui/console/views/device-add/manual/form/validation-schema.js

```diff
@@ -173,26 +173,74 @@ const sessionSchema = Yup.object({
 
 const macSettingsSchema = Yup.object({
   mac_settings: Yup.object().when(
-    ['_activation_mode', 'supports_class_b', 'supports_class_c'],
-    (mode, isClassB, isClassC, schema) =>
+    [
+      '_activation_mode',
+      'supports_class_b',
+      'supports_class_c',
+      '$hasRxDataRateOffset',
+      '$hasRxDelay',
+      '$hasRxDataRateIndex',
+      '$hasPingSlotDataRateIndex',
+      '$hasBeaconFrequency',
+      '$hasClassBTimeout',
+      '$hasClassCTimeout',
+      '$hasPingSlotFrequency',
+    ],
+    (
+      mode,
+      isClassB,
+      isClassC,
+      hasRxDelay,
+      hasRxDataRateOffset,
+      hasRxDataRateIndex,
+      hasPingSlotDataRateIndex,
+      hasBeaconFrequency,
+      hasClassBTimeout,
+      hasClassCTimeout,
+      hasPingSlotFrequency,
+      schema,
+    ) =>
       schema.shape({
+        resets_f_cnt: Yup.lazy(() => {
```

Comment:
@adriansmares it would be nice if you could skim through this validation schema to do a quick sanity check. :pray:

Reply:
Checked this - LGTM.